app/src/main/java/com/tpp/theperiodpurse/ui/legal/TermsScreen.kt

```diff
@@ -18,18 +18,19 @@ import androidx.navigation.compose.rememberNavController
 import com.tpp.theperiodpurse.R
 import com.tpp.theperiodpurse.ui.component.TopNavBar
 import com.tpp.theperiodpurse.ui.education.EducationBackground
+import com.tpp.theperiodpurse.ui.viewmodel.AppViewModel
 
 @SuppressLint("UnusedMaterialScaffoldPaddingParameter")
 @Composable
-fun TermsScreen(navController: NavHostController) {
+fun TermsScreen(navController: NavHostController, appViewModel: AppViewModel) {
```

Comment:
In dark mode the text in here cannot be seen, it's black text on dark background. I think composables in legal/CommonComponent needs to be updated and that should fix PrivacyScreen as well